core/templates/pages/exploration-editor-page/modal-templates/exploration-modify-translations-modal.component.html

```diff
@@ -0,0 +1,161 @@
+    <div *ngFor="let item of contentTranslations | keyvalue" class="content-translation-block">
+      <div class="content-translation-language">
+        <span>
+          <input type="checkbox" name="" id="">
+          <p>{{ getLanguageName(item.key) }}</p>
+        </span>
+        <i class="fas fa-pencil-alt"></i>
+      </div>
+      <div class="content-translation-value e2e-test-translation-{{ item.key }}">
+        <oppia-rte-output-display [rteString]="item.value.translation" class="translations-rte-output-display">
+        </oppia-rte-output-display>
+      </div>
+    </div>
+  </div>
+  <div *ngIf="contentTranslations && (contentTranslations | keyvalue).length === 0">
+    <p class="no-translations-found-text">No available translations found.</p>
+  </div>
+</div>
+<div class="modal-footer">
+  <button type="button" (click)="cancel()" class="btn modify-translations-cancel-button">Cancel</button>
+  <button type="button" (click)="confirm()" class="btn modify-translations-done-button">Done</button>
+</div>
+
+<style>
+  .modal-header {
+    justify-content: center;
+  }
+
+  .modal-title {
+    color: #00645c;
+    font-family: 'Capriola', 'Roboto', Arial, sans-serif;
+    font-size: 22px;
+    font-weight: 400;
+    height: 25px;
+  }
+
+  .modal-body {
+    padding: 24px;
+  }
+
+  .modal-body p {
+    font-size: 15px;
+  }
+
+  .modal-body .content-translations-container {
+    border-bottom: 1px solid rgba(0, 0, 0, 0.5);
+    border-top: 1px solid rgba(0, 0, 0, 0.5);
+    max-height: 330px;
+    overflow-y: auto;
+    padding-right: 8px;
+    padding-top: 14px;
+  }
+
+  .content-translations-container::-webkit-scrollbar {
+    width: 8px;
+  }
+
+  .content-translations-container::-webkit-scrollbar-track {
+    background: #e7e7e7;
+    border-radius: 5px;
+  }
+
+  .content-translations-container::-webkit-scrollbar-thumb {
+    background: #808080;
+    border-radius: 5px;
+    transition: background 0.5s ease;
+  }
+
+  .content-translations-container::-webkit-scrollbar-thumb:hover {
+    background: rgb(102, 102, 102);
+  }
+
+  .modal-body .content-translation-block {
+    margin-bottom: 12px;
+    padding-bottom: 11px;
+    padding-top: 4px;
+  }
+
+  .modal-body .content-translation-block:not(:last-child) {
+    border-bottom: 1px solid rgba(0, 0, 0, 0.5);
+  }
+
+  .modal-body .content-translation-language {
+    display: flex;
+    font-size: 18px;
+    justify-content: space-between;
+    margin-bottom: 10px;
+  }
+
+  .content-translation-language span {
+    display: flex;
+  }
+
+  .content-translation-language p {
+    font-size: 18px;
+  }
+
+  .content-translation-language input[type=checkbox] {
+    accent-color: #333;
+    margin-right: 8px;
+    width: 18px;
+  }
+
+  .content-translation-value {
+    border: 1px solid #ccc;
+    max-height: 115px;
+    overflow-y: scroll;
+    padding: 20px;
+  }
+
+  /* :ng-deep is used to style the inner elements of the modal.*/
```

Comment:
Is there a way to avoid ng-deep here?

Reply:
No, there are no alternative selectors suitable, only possible thing I can think of is setting a boolean in the RTE display component and adding a class to that RTE component accordingly. This code is still encapsulated within the context of our modal even with ng-deep though, so it shouldn't be a problem.

Reply:
Because we are changing the font size of an element inside a child component, we cannot do that without ng-deep unless we set custom variables to identify if the component is in the modal or not. I'm not sure why overriding font size is a problem here, it is used elsewhere in the codebase as well and it does not interfere with anything outside our modal.

Reply:
Sorry, just to clarify, I meant why do we need to change the font size for p tags in this context? What does it look like if we don't (trying to understand the need to do it..)

Reply:
Oh, it is just a visual thing, the font size is bigger otherwise and looks a bit cluttered.

Reply:
Per discussion with @Vir-8, the font size for some of the text content in the RTE looked bigger than the "instructions" in the modal, which looked odd, so Vir increased the size of the text content ("p" tags only.)

Vir to try other ways to avoid the use of ng-deep or increase the size of the instructions text.

Reply:
Done - changed the font size so it doesn't look odd without the ng-deep styling